.eslintrc

```diff
@@ -28,6 +28,7 @@
     "react/jsx-key": "warn",
     "react/no-string-refs": "warn",
     "react/prop-types": "warn",
+    "react/display-name": "off",
```

Comment:
I added this because it was giving me what I thought was an unreasonable error. (I was trying to create a cell renderer in SampleUploadTable, which involved a function that returned JSX. eslint interpreted this as a component class). Based on what I read about the rule, we don't seem to use it.

Reply:
I believe the rule just asks  you to defined the renderer as a named function vs. inline. 
In general, closures are considered bad practice, because this means that everytime you call it you are creating a new function. 
You should probably use a class method for the components.

Reply:
IMO, i would keep this rule.

Reply:
Okay, sounds good.